teuthology/nuke.py

```diff
@@ -400,19 +400,24 @@ def check_console(hostname):
         host=shortname,
         domain=config['ipmi_domain'])
     log.info('checking console status of %s' % cname)
-    if not console.check_status():
-        # not powered on or can't get IPMI status.  Try to power on
-        console.power_on()
```

Comment:
Well, I do change it in a later commit: I add a power cycle to help recover systems who may have hung. If you think that's a bad idea we should discuss